### PR TITLE
added component + moved checkboxWithTitle to atom section

### PR DESCRIPTION
**Pull request by GhaziCoorpacademy**
- moved checkbox-with-title component to atom section
- added new component: input-text-with-titeled-checkbox
input

**Detailed purpose of the PR**

**Result and observation**

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by GhaziCoorpacademy** [comment]
[ajout composant "add stars"](https://trello.com/c/Aomhgjpf/680-ajout-composant-add-stars)

File: packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        aria-label={checkBoxLabel}
+      />
+      <div className={style.inputText}>
+        <InputText
+          title={inputTextTitle}
+          value={inputTextValue}
+          placeholder={inputTextPlaceholder}
+          theme="coorpmanager"
+          description={inputTextDescription}
+          disabled={inputTextDisabled}
```
Comment (silouone) [inline packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js:40]: on ne lie pas directement l'etat disable au fait que la checkbox soit non coché ?

Comment (silouone) [reply packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js:40]: on peut gérer ça côté CM si on veut se laisse plus de maniabilité sur la désactivation mais pour l'instant on a pas de use case

Comment (DjamelSoualmi) [reply packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js:40]: add an aria-label for input plz

File: packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    </div>
+  );
+};
+
+InputTextWithTiteledCheckbox.propTypes = {
+  title: PropTypes.string,
+  name: PropTypes.string,
+  checked: PropTypes.bool,
+  checkBoxOnChange: PropTypes.func,
+  checkBoxLabel: PropTypes.string,
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js:54]:
```suggestion
  "aria-label": PropTypes.shape({
  input: PropTypes.string,
  checkbox: PropTypes.string
  }),
```

**Review by DjamelSoualmi — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js (hunk 1 above)
Comment (GhaziCoorpacademy) [reply packages/@coorpacademy-components/src/molecule/input-text-with-titeled-checkbox/index.js:40]: le composant ne prend pas d'aria-label.

**Review by silouone — APPROVED** [review]

**Review by DjamelSoualmi — APPROVED** [review]

**Pull request merged**